plugin/trino-kinesis/src/test/java/io/trino/plugin/kinesis/TestKinesisConfig.java

```diff
@@ -77,6 +79,7 @@ public void testExplicitPropertyMappings()
                 .setTableDescriptionLocation("/var/lib/kinesis")
                 .setDefaultSchema("kinesis")
                 .setHideInternalColumns(false)
+                .setUpdateInterval(new Duration(1, TimeUnit.MINUTES))
```

Comment:
This test will fail because the value is 600000ms. How about smaller one `500m` and `500, TimeUnit.MILLISECONDS`?